optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/config/PlannerBenchmarkConfig.java

```diff
@@ -209,7 +205,8 @@ public PlannerBenchmark buildPlannerBenchmark(SolverConfigContext solverConfigCo
         plannerBenchmarkResult.setAggregation(false);
         int parallelBenchmarkCount = resolveParallelBenchmarkCount();
         plannerBenchmarkResult.setParallelBenchmarkCount(parallelBenchmarkCount);
-        plannerBenchmarkResult.setWarmUpTimeMillisSpentLimit(defaultIfNull(calculateWarmUpTimeMillisSpentLimit(), 30L));
+        Long warmupTimeLimit = calculateWarmUpTimeMillisSpentLimit();
+        plannerBenchmarkResult.setWarmUpTimeMillisSpentLimit(Objects.isNull(warmupTimeLimit) ? 30L : warmupTimeLimit);
```

Comment:
Let's use "warmupTimeLimit == null" instead of Objects.isNull().
That isNull() method only exists to be used as a predicate for streams etc (see javadoc).